packages/api/src/routes/medical/dtos/network-entry-dto.ts

```diff
@@ -8,7 +9,7 @@ export function dtoFromHieDirectoryEntry(networkEntry: HieDirectoryEntry): Netwo
     name: networkEntry.name,
     oid: networkEntry.oid,
     zipCode: networkEntry.zipCode,
-    state: networkEntry.state,
+    state: normalizeUSStateForAddress(networkEntry.state ?? ""),
```

Comment:
Let's ship this for time sake, but I'd like to avoid using empty string if the state is required - I think it should either be optional or we filter out that entry as invalid.

Can you add this as a follow-up action/PR for this release, please?

Reply:
I'm happy to just add it here. You're saying state should never be null, and if it is we just drop the entity?